src/Microsoft.IdentityModel.Tokens.Saml/SamlTokenUtilities.cs

```diff
@@ -102,5 +104,60 @@ internal static IEnumerable<SecurityKey> GetKeysForTokenSignatureValidation(stri
             }
             return null;
         }
+
+        /// <summary>
+        /// Creates claims from  a dictionary.
+        /// </summary>
+        /// <param name="claimsCollection"> A dictionary that represents a set of claims.</param>
+        /// <returns> A collection of Claim objects created from the dictionary.</returns>
+        internal static IEnumerable<Claim> CreateClaimsFromDictionary(IDictionary<string, object> claimsCollection)
+        {
+            List<Claim> claims = null;
+            object value;
+            if (claimsCollection == null)
+                return claims;
+
+            claims = new List<Claim>();
+            foreach (string claimtype in claimsCollection.Keys)
+            {
+                if (claimsCollection.TryGetValue(claimtype, out value))
+                {
+                    string valueType = TokenUtilities.GetClaimValueTypeFromValue(value);
+
+                    if (value.GetType().Name == typeof(List<>).Name)
+                    {
+                        foreach (var item in (IList)value)
+                            claims.Add(new Claim(claimtype, item.ToString(), valueType));
+                    }
+                    else
+                    {
+                        claims.Add(new Claim(claimtype, value.ToString(), valueType));
+                    }
+                }
+            }
+
+            return claims;
+        }
+
+        /// <summary>
+        /// Merges IDictionary of claims and IEnumerable of claims.
```

Comment:
When referencing code in XML style comments it is standard to use a `<see cref="{symbolName}"/>`

Reply:
Added symbol referencing.